partridge/readers.py

```diff
@@ -13,6 +14,22 @@
 '''Public'''
 
 
+def get_filtered_feed(path, filters, config=None):
+    '''Multi-file feed filtering'''
+    def reachable_trip_ids(filename, column_filters):
+        root_config = config_for_root(filename)
```

Comment:
Need to accept a custom config here, to allow filtering on non-standard files. `config_for_root` might need to change or just go away and use `reroot_graph` directly.